includes/Modules/Thank_With_Google.php

```diff
@@ -410,9 +423,9 @@ function ( $host ) {
 					$is_connected            = $this->is_connected();
 					$setup_transient         = $this->transients->get( self::TRANSIENT_SETUP_TIMER );
 					if ( $is_connected && empty( $supporter_wall_sidebars ) && ! $setup_transient ) {
```

Comment:
The `is_connected` state doesn't really belong here – we shouldn't be calling this endpoint before the module is connected.

Reply:
Updated.